bin/studio-cli.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# The least terrible way to resolve a symlink to its real path.
+function realpath() {
+  /usr/bin/perl -e "use Cwd;print Cwd::abs_path(@ARGV[0])" "$0";
+}
+
+CONTENTS="$(command dirname "$(command dirname "$(command dirname "$(realpath "$0")")")")"
+BINARY_NAME="$(TERM=dumb command ls "$CONTENTS/MacOS/")"
+ELECTRON="$CONTENTS/MacOS/$BINARY_NAME"
+CLI="$CONTENTS/Resources/app/cli/index.js"
+
+ELECTRON_RUN_AS_NODE=1 "$ELECTRON" "$CLI" "$@"
+
+exit $? 
```

Comment:
Nit-pick: this exit command is redundant, as scripts implicitly return the exit code of their last invocation. But I would go further here and propose using `exec` to _replace_ the shell with the Electron program rather than spawn it as a child process. 

Unless there are specific reasons to keep the shell running, I'd say `exec` is not only more economical but can help minimise surprises around job control [based on my experience with wp-now](https://github.com/WordPress/playground-tools/issues/220).

```sh
ELECTRON_RUN_AS_NODE=1 exec "$ELECTRON" "$CLI" "$@"
```

Reply:
Interesting. Using `exec` seems like the way to go 👍 Good catch.

We expect the CLI will run more and more long-running commands, so if not in the first iteration, I'm sure someone would eventually have run into job control issues similar to what you described.